tests/Oryx.Integration.Tests/LocalDockerTests/PythonEndToEndTests.cs

```diff
@@ -560,6 +560,86 @@ public async Task CanBuildAndRun_Tweeter3App()
                 });
         }
 
+        [Theory]
+        [MemberData(nameof(TestValueGenerator.GetPythonVersions), MemberType = typeof(TestValueGenerator))]
+        public async Task CanBuildAndRun_ShapelyFlaskApp_UsingVirtualEnv(string pythonVersion)
+        {
+            // Arrange
+            var appName = "shapely-flask-app";
+            var virtualEnvName = "antenv";
+            var hostDir = Path.Combine(_hostSamplesDir, "python", appName);
+            var volume = DockerVolume.Create(hostDir);
+            var appDir = volume.ContainerDir;
+            var portMapping = $"{HostPort}:{ContainerPort}";
+            var script = new ShellScriptBuilder()
+                .AddCommand("pip install gunicorn")
+                .AddCommand("pip install flask")
+                .AddCommand("pip install shapely")
+                .AddCommand($"cd {appDir}")
+                .AddCommand($"oryx -appPath {appDir} -bindPort {ContainerPort} -virtualEnvName {virtualEnvName}")
```

Comment:
You need not specify virtual environment name here again as we write to the manifest file about it and which the runtime image reads